### ISSUE TITLE
No error on repeated property

### ISSUE THREAD

**Issue by danielgordon10** [microsoft/pyright#1553]
**Describe the bug**
The checker correctly identifies when a function is redefined, but does not do so for properties.

**To Reproduce**
```python
class Obj():
    def __init__(self):
        self._property = 0

    @property
    def prop(self):
        return  self._property

    @property
    def prop(self):
        return  self._property
```

**Expected behavior**
An error similar to one which exists already on redefined functions such as `267:9 - error: Method declaration "prop" is obscured by a declaration of the same name`

**Comment by erictraut** [comment]
Thanks for the suggestion. This will be included in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.117, which I just published. It will also be included in the next release of pylance.

**Comment by danielgordon10** [comment]
Awesome, thanks for the fast turnaround.
